types/index.d.ts

```diff
@@ -696,6 +696,8 @@ declare namespace postgres {
   interface TransactionSql<TTypes extends Record<string, unknown> = {}> extends Sql<TTypes> {
     savepoint<T>(cb: (sql: TransactionSql<TTypes>) => T | Promise<T>): Promise<UnwrapPromiseArray<T>>;
     savepoint<T>(name: string, cb: (sql: TransactionSql<TTypes>) => T | Promise<T>): Promise<UnwrapPromiseArray<T>>;
+
+    prepare<T>(name: string): Promise<UnwrapPromiseArray<T>>;
```

Comment:
The `T` parameter is completely useless here (not used in parameters or in return type) so it should be removed to avoid confusing users

```suggestion
    prepare(name: string): Promise<void>;
```